generated/provider_dependencies.json

```diff
@@ -526,7 +526,7 @@
     "deps": [
       "apache-airflow>=2.10.0",
       "pydantic>=2.10.2",
-      "retryhttp>=1.2.0"
+      "retryhttp>=1.2.0,!=1.3.0"
```

Comment:
Should we do <1.3.0, in case they release a new version that doesn't fix it?

Reply:
Long term yeah

Reply:
The fact that we are going to revert this anyways after they fix it makes me think that'd be better

Reply:
@jedcunningham did that, look any better?